bouncer/__init__.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import pyramid.config
+
+
+def settings():
```

Comment:
This is only ever used once though, and like this: `settings=settings()`. Is there really potential for confusion? I kind of like the simpler name.